src/plugins/console/server/lib/spec_definitions/json/generated/ml.start_trained_model_deployment.json

```diff
@@ -1,14 +1,20 @@
 {
   "ml.start_trained_model_deployment": {
     "url_params": {
-      "timeout": ""
+      "cache_size": "",
+      "number_of_allocations": 0,
+      "threads_per_allocation": 0,
+      "priority": "",
+      "queue_capacity": 0,
+      "timeout": "",
+      "wait_for": ""
     },
     "methods": [
       "POST"
     ],
     "patterns": [
       "_ml/trained_models/{model_id}/deployment/_start"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-start-trained-model-deployment.html"
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/start-trained-model-deployment.html"
```

Comment:
Should this stay as `current`?

Reply:
I noticed the same thing, the changed links are valid links I've tested they point to the right documentation. 

I spoke to the docs team, they are aware of the mixed usage of both **current** and **master** and expect the URLs to be rationalised in a future change. 

These files were generated directly from the Elasticsearch Rest API spec, I think it is best not to deviate from that spec by making edits here.

Reply:
Thanks for the explanation!

> I spoke to the docs team, they are aware of the mixed usage of both current and master and expect the URLs to be rationalised in a future change.

As far as I understand, `master` is always going to point to the latest, unreleased version, whereas `current` is going to point to the latest release. So while the docs currently are correct, that might not always be the case once the 8.7 branch is cut. 

That said, if the docs team is aware of the mixed usage, I'm OK to leave as-is for now.

Reply:
I tested this a bit and it seems like irrespective of whether the spec says current or master it's being mapped to the appropriate docs version, perhaps via this code?: https://github.com/elastic/kibana/blob/main/src/plugins/console/public/application/containers/editor/legacy/console_menu_actions.ts#L31. In the long run, the console ought to use the [doc link service](https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts) or the Elasticsearch client specs [#130553](https://github.com/elastic/kibana/issues/130553), since our URLs will likely be completely different in the new docs system.